Menus/MainMenu/MainMenu.cs

```diff
@@ -121,6 +109,25 @@ void Start()
 			UpdateToolButtons();
 		}
 
+		public void SetVisible(bool visible, bool temporary = false)
+		{
+			if (m_Visible != visible)
+			{
+				m_Visible = visible;
+				if (m_MainMenuUI)
+				{
+					m_MainMenuUI.visible = visible;
+					if (!temporary)
+						SendVisibilityPulse();
+				}
+			}
+		}
+
+		public bool GetVisible()
```

Comment:
Either way, the issue is that we're using methods instead of properties, which I don't think can be helped. @dunity, would you still like me to turn GetVisible into a set-only property? Should we leave it as it is?